packages/host/src/components/Login/hooks.ts

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
```

Comment:
I do like having the state next to the component.. but it was getting too big!
Slightly worried about having effects in a separate file, and the confusion that might cause.. only slightly though